### ISSUE TITLE
Train cannot begin?

### ISSUE THREAD

**Issue by wsqat** [yulunzhang/RCAN#53]
Why?

parser.add_argument('--n_train', type=int, default=15000,#800
                    help='number of training set')
parser.add_argument('--n_val', type=int, default=10,
                    help='number of validation set')
parser.add_argument('--offset_val', type=int, default=15000,#800

```
[Epoch 15]	Learning rate: 1.00e-4

  0%|                                                    | 0/10 [00:00<?, ?it/s]
Evaluation:
100%|███████████████████████████████████████████| 10/10 [00:42<00:00,  4.16s/it][Set5 x4]	PSNR: 37.509 (Best: 37.509 @epoch 1)
Total time: 42.01s

[Epoch 16]	Learning rate: 1.00e-4

  0%|                                                    | 0/10 [00:00<?, ?it/s]
Evaluation:
100%|███████████████████████████████████████████| 10/10 [00:41<00:00,  4.16s/it][Set5 x4]	PSNR: 37.509 (Best: 37.509 @epoch 1)
Total time: 41.92s
```

**Comment by wsqat** [comment]
@yulunzhang Whether the train image numbers is too large? when I change 1000, that's can run.

**Comment by wsqat** [comment]
@MaFuyan Hi?

**Comment by yulunzhang** [comment]
Hi,

You can change the Line 15 of RCAN/RCAN_TrainCode/code/data/div2k.py to self.repeat = 1
